go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/lazyledger/rsmt2d
+
+go 1.14
+
+require (
+	github.com/NebulousLabs/errors v0.0.0-20181203160057-9f787ce8f69e // indirect
+	github.com/NebulousLabs/fastrand v0.0.0-20181203155948-6fb6489aac4e // indirect
+	github.com/NebulousLabs/merkletree v0.0.0-20181203152040-08d5d54b07f5
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	// only needed if built with go build -tags leopard
+	github.com/liamsi/go-leopard v0.0.0-20200522120806-b2bdbf8f6401
```

Comment:
This needs updating if we move `go-leopard` to the LL org. I probably should do a release too (instead of referencing via commit).